obu/app/database/v2x_db.cc

```diff
@@ -32,6 +34,7 @@ namespace v2x {
 void V2xDb::V2xDbInit() {
   spat_ = new V2xTable(true, 5);
   map_ = new V2xTable(false, 5);
+  ssm_ = new V2xTable(true, 5);
```

Comment:
Quick question, why not manage them by `std::unique_ptr`?

Reply:
At present it is just to keep the style consistent with the existing code. This is a good suggestion, and we will first update  on the main line.